src/Symfony/Component/Ldap/Adapter/ExtLdap/EntryManager.php

```diff
@@ -139,4 +158,21 @@ public function applyOperations(string $dn, iterable $operations): void
             throw new UpdateOperationException(sprintf('Error executing UpdateOperation on "%s": "%s".', $dn, ldap_error($this->getConnectionResource())));
         }
     }
+
+    /**
+     * Parse the RDN out of an entry's DN for use in a move operation.
+     *
+     * @param Entry $entry to parse the RDN from
```

Comment:
This can be removed as it does not really add any useful information. I would even go as far as removing the whole docblock here (especially as this is a private method).

Reply:
Addressed, thanks! I meant to include something useful here, but I seem to have not gotten around to it and can't remember what that might have been at this point.